client/tests/acceptance/user-sees-projects-of-all-types-test.js

```diff
@@ -27,7 +27,10 @@ module('Acceptance | user sees projects of all types', function(hooks) {
     assert.equal(findAll('[data-test-view-pas]').length, 5);
   });
 
-  skip('Page should display non-assigned message if no projects');
+  test('Page should display non-assigned message if no projects', async function(assert) {
+    await visit('/projects');
+    assert.ok(find('[data-test-not-assigned]'));
+  });
```

Comment:
At first, I wondered where the data setup was, but then I realized it implicitly provides no projects because none get created!

If I realllly wanted to get pick, though, I might've tried this:

```
this.server.createList('project', 0);
```

I'm sure mirage doesn't allow for that, though, haha.

Reply:
ohhh yes. I'm always a fan of being explicit. hmmmmm interesting idea